docs/en/sdk-manual/http-api.md

```diff
@@ -0,0 +1,215 @@
+|------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|
+| Content-Type     | YES      | application/json; charset=utf-8                                                                                                     | Request Type                                                                      |
+| X-Forwarded-For  | NO       | [101.188.67.134](https://whatismyipaddress.com/page/36)                                                                             | Source IP address                                                                 |
+| User-Agent       | NO       | Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Mobile Safari/537.36 | User-Agent，The browser will bring it by default when requesting                   |
+| cookie           | NO       | your auth cookie                                                                                                                    | Customize the verification cookie generated by OIDC for interface authentication  |
+
+### Request query parameters
+
+| Parameter name           | Required  | Example             | Description                                                                                                                                                |
+|--------------------------|-----------|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| appId                    | YES       | test_app            | The app id when creating the application can be obtained from the Clickstream Web Console                                                                  |
+| platform                 | NO        | Android/iOS/Web/... | Distinguish between platforms                                                                                                                              |
+| event_bundle_sequence_id | NO        | 1                   | Request sequence number, an incrementing integer starting from 1                                                                                           |
+| hashCode                 | NO        | 478acd09            | The first 8 digits of the sha256 calculation result of the request body string are used by the server to verify whether the request body has been modified |
+| compression              | NO        | gzip                | Request body compression method. Currently only gzip is supported. Not passing this field means no compression                                             |
```

Comment:
-> Request body compression method. Currently, only gzip is supported. Keeping it absent means no compression

Reply:
changed to this.